utils/generate-playbook

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python3
+import argparse
+from argparse import RawTextHelpFormatter
+import re
+import sys
+from os.path import abspath, dirname, join
+from ruamel import yaml
+from operator import itemgetter
+from collections import defaultdict
+
+# Import common_koji from the local "module_utils" directory.
+working_directory = dirname(abspath((__file__)))
+module_utils_path = join(dirname(working_directory), 'module_utils')
+if module_utils_path not in sys.path:
+    sys.path.insert(0, module_utils_path)
+import common_koji  # NOQA: E402
+
+
+DESCRIPTION = """
+This tool queries Koji for all the tags and their settings.
+It then writes an Ansible playbook to STDOUT.
+
+Use this tool to quickly get started with koji-ansible, or to transfer
+a configuration from one Koji instance to another.
+"""
+
+
+def interrogate(session, tags):
+    """
+    Interrogate Koji for all information about these tags.
+
+    Quickly get all the information we need from Koji with as few XML-RPC
+    calls as possible.
+    """
+    # Note: This client multiCall method bombs out after about 35 minutes for
```

Comment:
You could use "itercall", which is a wrapper around multiCall that makes it easier to use, and also splits huge calls into smaller chunks of configurable size. See Koji pull request: https://pagure.io/koji/pull-request/116

Reply:
Thank you for pointing this out. I switched to the new multi-call interface in Koji 1.18 and I set `batch=100`.